develdocs.phtml

```diff
@@ -68,11 +68,11 @@ include("externals/menu.phtml");
 </div>
 <p class="justify"><?php echo T_("Get a copy of the documentation source as described above and start making changes. When you are satisfied with your changes you can create a patch by running the following command in the base directory of the documentation (usually gnucash-docs, unless you renamed it):");?></p>
 <pre class="command">git diff > my-patch-name.patch</pre>
-<p class="justify"><?php echo T_("Next attach your patch to a bug report against the proper component for the Documentation product in <a href='https://bugs.gnucash.org/'><span class=\"gnucash\">GnuCash</span>' bug database</a>. See also <a href='https://wiki.gnucash.org/wiki/Bugzilla'>GnuCash' wiki page on bugzilla</a> for more details about patch submission.");?></p>
+<p class="justify"><?php echo T_("Next, attach your patch to a bug-report against the proper component for the <b>Documentation</b> product in the <a href='https://bugs.gnucash.org/'><span class=\"gnucash\">GnuCash</span>' bug database</a>. See also <a href='https://wiki.gnucash.org/wiki/Bugzilla'>GnuCash' wiki page on Bugzilla</a> for more details about patch submission.");?></p>
```

Comment:
I don't believe that "bug report" is hyphenated.

Reply:
Also, change:

"the proper component for the <b>Documentation</b> product"

To:

"the <b>Documentation</b> component"

Reply:
See
- [Browse](https://bugs.gnucash.org/describecomponents.cgi)
- [Components for Documentation](https://bugs.gnucash.org/describecomponents.cgi?product=Documentation)

Reply:
Another note. This section is - like several pages in the wiki - outdated. Today we want:

1. If possible: Pull Requests at Github,
2. else: patches at bugzilla.
3. No longer recommend as they are often forgotten: patches via gnucash-devel.

Reply:
I see your point in re: terminology. Perhaps refer to Documentation as a section? As in:

"the proper component under the Documentation section in the GnuCash bug database"

Reply:
Bugzilla groups bugs by **product** and **component**. The **products** in GnuCash's BZ are `Bugzilla`, `Documentation`, `GnuCash`, `Packaging`, and `Website`. See the [New Bug Page](https://bugs.gnucash.org/enter_bug.cgi). It might seem odd to call Documentation a product, but calling it something else would be confusing.

Reply:
They are not consistent. In Browse they use **product category**.

Reply:
> "the proper component under the Documentation section in the GnuCash bug database"

3 steps backwards to execute sounds confusing.
How about  converting the sentence in a numbered list:

1. open "https://bugs.gnucash.org/enter_bug.cgi"
2. select product "Documentation"
3. select the proper component …

Reply:
> They are not consistent. In Browse they use product category.

But only there. It's just *product* everywhere else, and nowhere is it section.

Reply:
Given the bugzilla nomenclature, the original should be used. Sorry for the confusion.